docs/data/future-datamodel.md

```diff
@@ -111,119 +123,147 @@ and may be related to various entities within the hierarchy.
 - `pid`: A unique persistent identifier (for now ARK URL) for the project.
-- `shortcode`: The project's DSP short code.  
-  4 characters hexadecimal, upper case.
+- `shortcode`: The project's DSP short code, internal only.  
+  4 characters hexadecimal, upper case.  
 - `status`: The status of the project.  
   Either "Ongoing" or "Finished".
 - `name`: The name of the project.
 - `shortDescription`: A short text to be displayed as a teaser.  
   Maximum length: 200 characters (all including).
-- `description`: The description of the project.
+- `description`: The full description of the project.
 - `startDate`: The start date of the project.
 - `endData`: The end date of the project.
-- `url`: The URL of the project.
+- `url`: The URL to the web presence of the project.  
+  The first URL should point to where the data is available,
+  the second, optional URL may point to the project website.
 - `howToCite`: How to cite the project.  
   If not provided, we use the standard form `<contributors> (<year>). <project name> [Database]. DaSCH. <ARK>`.
-- `accessRights`: The access rights of the project.
+- `accessRights`: The access rights of the project.  
+  Literal "Full Open Access", "Open Access with Restrictions", "Embargoed Access", "Metadata only Access".
+  If the project is embargoed, the metadata is only available on the project level.
+  Access rights define to what extent the project data is available to access in the DPE.
 - `legalInfo`: Legal information about the project.
   Calculated from Datasets. Can _not_ be specified explicitly on the project.
 - `dataManagementPlan`: A data management plan of the project.
   String or URL, use "not accessible" if not available to us.
 - `datasets`: A list of dataset identifiers that make up the project data.
+- `records`: A list of record identifiers that make up the project data.
 - `keywords`: A list of keywords describing the project.
 - `disciplines`: A list of disciplines the project is related to.
-- `temporalCoverage`: A list of temporal coverages of the project.
-- `spatialCoverage`: A list of spatial coverages of the project.
+- `temporalCoverage`: A list of epoches or time periods the project is related to.
+- `spatialCoverage`: A list of references to spatial entities (Places, Regions, etc.) the project is related to.
 - `attributions`: A list of attributions defining what roles people/organizations have in the project.  
   Manually entered, as there may be people who don't have authorship, like reviewers, organizers, etc.
 - `abstract`: An abstract of the project.
-- `contactPoint`: A person or organization responsible for the project.
+- `contactPoint`: A person or organization serving as first contact for the project.
 - `publications`: A list of publications related to the project.
-- `grants`: A list of grants related to the project.
+- `funding`: Either a string ("No funding") or a list of grants received by the project.
```

Comment:
IMHO representing the absence of a value with a specific string may be the source of future problems. 
E.g. developers and users need to now the very specific string value and may not make the most common mistake - a typo. In order to prevent this usually code must be written and maintained.

Why not making the funding optional?

Reply:
In the long term, we will need a way to represent generic "no formal funding received" (which applied to projects that were self-funded or where we do not have other information) and "funded through a structural position at an institution" (e.g. a PhD project of someone who had a "Assistenzstelle") which also isn't formally funded through a  "grant".  
If we model this as the absence of a grant, we can not make this distinction.

And the fact that this is called "string" here, of course does not mean that we have to (or should!) implement this as a string. I would think of it as two enum variants.  
But in the JSON which we use as our intermediary metadata format, there is no real way of representing enums.